packages/lib/src/vis/utils.ts

```diff
@@ -342,3 +342,16 @@ export function getCameraFOV(camera: Camera): {
 
   return { topRight, bottomLeft };
 }
+
+export function getUnprojectedPoint(
+  event: PointerEvent | WheelEvent,
+  size: Size,
+  camera: Camera
+): Vector3 {
+  const { offsetX: x, offsetY: y } = event;
+  const { width, height } = size;
```

Comment:
The size is supposed to be retrieved from `useThree`. 

Would it be possible to retrieve it from the `event` ? It contains the `innerWidth` and `innerHeight` of the element on which the event is attached, right (hence the canvas) ?